app/controllers/organizations_controller.rb

```diff
@@ -52,6 +60,7 @@ def organization_params
       :name,
       :phone,
       :url,
+      :status
```

Comment:
In Ruby, you can actually put an extra comma at the end of lists like this.
It's a bit weird, but it's good to use trailing commas so that we can re-order the lines however we like.